apps/nightscout/nightscout.go

```diff
@@ -0,0 +1,25 @@
+// Package nightscout provides details for the Nightscout applet.
+package nightscout
+
+import (
+	_ "embed"
+
+	"tidbyt.dev/community/apps/manifest"
+)
+
+//go:embed nightscout.star
+var source []byte
+
+// New creates a new instance of the Nightscout applet.
+func New() manifest.Manifest {
+	return manifest.Manifest{
+		ID:          "nightscout",
+		Name:        "Nightscout",
+		Author:      "Jeremy Tavener",
+		Summary:     "Shows Nightscout CGM Data",
+		Desc:        "Displays Continous Glucose Monitoring (CGM) data from the Nightscout Open Source project (https://nightscout.github.io/).",
```

Comment:
One thing that makes me pretty nervous is the failure cases around Tidbyt today. If we fail to render an app, the Tidbyt will happily continue to serve an instance from it's local cache forever until it gets an updated copy. It's happened before with the clock where we continue to show a stale time which is really a poor use case. We're working to be able to set a freshness setting for each app to be able to determine how stale it can be, but we don't have it just yet.

Is this app ok to be stale? Should we put a warning in the description saying it could be stale if Tidbyt has problems? Do we need to wait until we get the ability to only serve fresh renders?

Reply:
I could rework the UI to also display a "Last Updated" field.  Then if for some reason it's using a local cached version, the information is still accurate (albeit old).

Reply:
Ohhh, I like that a lot. Displaying last updated would make me feel a lot more comfortable